### PR TITLE
login section added

### PR DESCRIPTION
**Pull request by smruti2705**

### CODE REVIEW

File: src/Components/Login/Login.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import React, {useState} from 'react';
+import login from './login.module.scss';
```
Comment (HarshhFarkiya) [inline src/Components/Login/Login.js:2]: can we just write one import statement for all the images to import?

**Comment by HarshhFarkiya** [comment]
Try to make the design responsive

**Comment by HarshhFarkiya** [comment]
Make the section responsive, and also add something for the small devices.

File: src/Components/Login/Login.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    }
+}
+  return (
+    <div className={login.loginpage}>
+        <form className={login.form}>
+      <img src={logo1} alt='logo' className={login.logo}/>
+      <p className={login.head}>Hello, Welcome back!</p>
+      <div className={login.emailinfo} >
+        <img src={email} alt='email' className={login.emailimg}/>
+        <input
```
Comment (HarshhFarkiya) [inline src/Components/Login/Login.js:32]: take the input in useState and add function to handle that input and for now print it on console

File: src/Components/Login/Login.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+          type='text'
+          className={login.email}
+          placeholder='Email'>
+        </input> 
+      </div> 
+      <div className={login.passinfo} >
+        <img src={lock} alt='password' className={login.passimg}/>
+        <input
```
Comment (HarshhFarkiya) [inline src/Components/Login/Login.js:40]: make the fields as required fields

Comment (DeepakkPatil) [reply src/Components/Login/Login.js:55]: done

File: src/Components/Login/Login.js (hunk 2 above)
Comment (DeepakkPatil) [reply src/Components/Login/Login.js:45]: done

**Pull request closed without merging**